lib/commands/base.rb

```diff
@@ -4,23 +4,26 @@
 
 module Commands
   class Base
+    attr_reader :config_directory, :service, :stack, :verbose
```

Comment:
Do these need to be public?

Reply:
Ah, no - it got a bit mixed up in the rebase. Private again.